src/credential-status/status-token.ts

```diff
@@ -0,0 +1,159 @@
+import AbortController from 'abort-controller'
+import fetch from 'node-fetch'
+import { cborDecode, cborEncode } from '../cbor'
+import { Tag } from '../cbor/cbor-x'
+import type { CoseKey } from '../cose'
+import { CoseType, CoseTypeToTag, Mac0, Sign1 } from '../cose'
+import { CWT } from '../cwt'
+import type { StatusArray } from './status-array'
+import { StatusList } from './status-list'
+
+interface CWTStatusTokenOptions {
+  claimsSet: {
+    statusArray: StatusArray
+    aggregationUri?: string
+    expirationTime?: number
+    timeToLive?: number
+  }
+  type: CoseType
+  key: CoseKey
+}
+
+interface CWTStatusTokenVerifyOptions {
```

Comment:
Can you change this everywhere `CWT` is used?
```suggestion
interface CwtStatusTokenVerifyOptions {
```